ngraph/core/src/function.cpp

```diff
@@ -405,4 +405,18 @@ void Function::remove_result(const std::shared_ptr<op::Result>& result)
         m_results.end());
 }
 
+void Function::add_parameters(const ParameterVector& params)
+{
+    m_parameters.insert(m_parameters.end(), params.begin(), params.end());
+}
+
+void Function::remove_parameter(const std::shared_ptr<op::Parameter>& param)
```

Comment:
Why do we need to have these methods?

Reply:
To be able to remove parameters after converting to variables after Low-latency transformation. 
The second reason is symmetry with results API.

Reply:
It would be nice to have an external transformation to replace Parameter to Constant with some value, so this API would be useful here
Also, the user can also replace Parameter for some other nodes, e.g. ReadValue